pkgs/development/python-modules/importlib-resources/2.nix

```
@@ -0,0 +1,37 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, setuptools_scm
+, toml
+, importlib-metadata
+, typing
+, singledispatch
+, python
+}:
+
+buildPythonPackage rec {
+  pname = "importlib_resources";
+  version = "3.3.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0ed250dbd291947d1a298e89f39afcc477d5a6624770503034b72588601bcc05";
+  };
+
+  nativeBuildInputs = [ setuptools_scm toml ];
```

```suggestion
  nativeBuildInputs = [ setuptools-scm ];
```

toml is no longer required because I added it to setuptools-scm.